website/homepage/render_html.py

```diff
@@ -43,7 +43,7 @@ def render_index():
 
 guides = []
 for guide in sorted(os.listdir(GRADIO_GUIDES_DIR)):
-    if "readme" in guide.lower():
+    if guide.lower() in ["getting_started.md", "readme.md"]:
```

Comment:
@abidlabs why did you add `getting_started.md` here? was that intentional? It stops `/getting_started` from getting rendered. Note it already gets skipped in the main guides page if that's what you meant

Reply:
Ah yes it looks like that was a mistake on my part